website/docs/r/app_service_environment.html.markdown

```diff
@@ -78,6 +80,10 @@ resource "azurerm_app_service_environment" "example" {
 
 * `resource_group_name` - (Optional) The name of the Resource Group where the App Service Environment exists. Defaults to the Resource Group of the Subnet (specified by `subnet_id`).
 
+* `version` - (Optional) The Version of the Application Service Environment. Possible values are `ASEV2` and `ASEV3`. Defaults to `ASEV2`.
```

Comment:
thinking about the deprecation/removal path for v2 down the line here: would this make more sense to split out into a `_v3` resource?

Reply:
The preview currently "fits" within the existing resource, and my original thought was to be able to deprecate fields over time as Azure promotes V3 to the default option.  That said, there's enough difference in implementation to warrant a separate resource. I'll pick up with you offline to discuss before assigning this PR to a release.